.gitlab-ci.yml

```diff
@@ -72,31 +94,87 @@ workflow:
 .start-substrate-contracts-node-rand-extension:      &start-substrate-contracts-node-rand-extension
   - substrate-contracts-node-rand-extension --tmp --dev -lruntime::contracts=debug > /tmp/substrate-contracts-node-rand-extension.log 2>&1 &
 
+.shutdown-substrate-contracts-node:                  &shutdown-substrate-contracts-node
+  - pkill -f "substrate-contracts-node --tmp --dev"
+
+.shutdown-substrate-contracts-node-rand-extension:   &shutdown-substrate-contracts-node-rand-extension
+  - pkill -f "substrate-contracts-node-rand-extension --tmp --dev"
+
+
+# needed vars have to be "exported" on a earlier stage
+parent-vars:
+  stage:                                             declare-vars
+  <<:                                                *kubernetes-env
+  script:
+    - BRANCH="master"
```

Comment:
is this a current repo's branch? Where is it used? From what I see it's only `REDIS_SIZES_KEY`.
Better have a more distinctive name.

Reply:
AFAIU this is [the default branch](https://github.com/paritytech/ink-waterfall/commit/9928941b0ffe54ccc06f6a8f7cebb81ae5d7fb46) if a pipeline wasn't triggered by a upstream.